src/stepFunctions/commands/visualizeStateMachine/aslVisualizationCDKManager.ts

```diff
@@ -36,7 +36,7 @@ export class AslVisualizationCDKManager extends AbstractAslVisualizationManager<
         try {
             await this.cache.updateCache(globalStorage)
 
-            const textDocument = await vscode.workspace.openTextDocument(templateUri)
+            const textDocument = await vscode.workspace.openTextDocument(templateUri.with({ fragment: '' }))
```

Comment:
Something is weird with `tryOpenDocument` on C9. Something is getting converted to `undefined` if the fragment is set. Not sure if it's the version or just C9. I'll add a test.